### ISSUE TITLE
Doctest failure

### ISSUE THREAD

**Issue by DanBurton** [haskell-servant/servant#698]
As seen on the nightly stackage build.

```
> /tmp/stackage-build14/servant-0.10$ dist/build/doctests/doctests
### Failure in /tmp/stackage-build14/servant-0.10/test/Servant/Utils/LinksSpec.hs:84: expression `apiLink (Proxy :: Proxy WrongContentType)'
expected: ...
          ...Could not deduce...
          ...
 but got:
          <interactive>:27:1: error:
              <E2><80><A2> OctetStream expected in list '[JSON]
              <E2><80><A2> In the expression: apiLink (Proxy :: Proxy WrongContentType)
                In an equation for <E2><80><98>it<E2><80><99>:
                    it = apiLink (Proxy :: Proxy WrongContentType)
Examples: 141  Tried: 138  Errors: 0  Failures: 1
```

**Comment by DanBurton** [comment]
Since this looks benign I've simply marked servant as "expected test failure" on stackage until this is resolved.

**Comment by phadej** [comment]
This should be fixed, but let's see after `servant-0.11` reaches the nightly and we enable tests, and they pass.

**Comment by bergmark** [comment]
Passed in the latest nightly build so this can be closed.
